src/components/base/BaseSelect.vue

```diff
@@ -0,0 +1,59 @@
+<script setup lang="ts">
+interface Props {
+  options: string[];
+  modelValue: string;
+}
+const props = withDefaults(defineProps<Props>(), {
+  options: null,
+  modelValue: "",
+});
+</script>
+
+<template>
+  <div class="base-dropdown">
```

Comment:
It's nice to keep class names matching file names so here could be renamed to `base-select`